### PR TITLE
LPS-178924 Add integration tests to ensure that is possible enable and disable categorization and comments in modifiable system objects

### PR DESCRIPTION
**Pull request by seltong**
Related issues:

- [LPS-178220](https://issues.liferay.com/browse/LPS-178220)
- [LPS-178222](https://issues.liferay.com/browse/LPS-178222)
- [LPS-178225](https://issues.liferay.com/browse/LPS-178225)
- [LPS-178809](https://issues.liferay.com/browse/LPS-178809)
- [LPS-178924](https://issues.liferay.com/browse/LPS-178924)

### CODE REVIEW

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionException.java [hunk 1]
```diff
@@ -21,19 +21,16 @@
...
 		super(msg);
 	}
 
-	public ObjectDefinitionException(String msg, Throwable throwable) {
-		super(msg, throwable);
-	}
+	public static class MustBeModifiable extends ObjectDefinitionException {
+
+		public MustBeModifiable() {
+			super("Require a modifiable object definition");
```
Comment (gabrielwas) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionException.java:31]: Change to "A modifiable object definition is required"

**Review by gabrielwas — CHANGES_REQUESTED** [review]
Check the comment, please @seltong

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:object

**Review by gabrielwas — APPROVED** [review]
LGTM :)

**Pull request closed without merging**